django_netjsonconfig/tests/test_controller.py

```diff
@@ -28,7 +28,7 @@ def _check_header(self, response):
     def test_checksum(self):
```

Comment:
I don't see new tests for the new views, that's not good. We need to automatically test every feature we add, in this case the VPN config views, because otherwise if one day someone inadvertently breaks something in the vpn views we won't find out until the bug will come out in some production environment.

Could you please add the tests for the new views? You can reuse the same structure of the current tests for device checksum and download config.